callbacks.go

```diff
@@ -116,7 +116,7 @@ func (exec *sidecarExecutor) LaunchTask(driver executor.ExecutorDriver, taskInfo
 
 	// Start the container
 	log.Info("Starting container with ID " + cntnr.ID[:12])
-	err = exec.client.StartContainer(cntnr.ID, containerConfig.HostConfig)
+	err = exec.client.StartContainer(cntnr.ID, nil)
```

Comment:
starting container with HostConfig was deprecated since v1.10 and removed in v1.12